designer/client/outputs/notify-edit.tsx

```diff
@@ -47,42 +51,54 @@ class NotifyEdit extends Component<Props, State> {
 
     return (
       <div className="govuk-body">
-        <div className="govuk-form-group">
-          <label className="govuk-label" htmlFor="template-id">
-            Template ID
-          </label>
-          <input
-            className="govuk-input"
-            name="template-id"
-            type="text"
-            required
-            defaultValue={templateId}
-            step="any"
-          />
-        </div>
-        <div className="govuk-form-group">
-          <label className="govuk-label" htmlFor="api-key">
-            API Key
-          </label>
-          <input
-            className="govuk-input"
-            name="api-key"
-            type="text"
-            required
-            defaultValue={apiKey}
-            step="any"
-          />
-        </div>
-        <div className="govuk-form-group">
+        <Input
+          id="template-id"
+          name="template-id"
+          label={{
+            className: "govuk-label--s",
+            children: ["Template ID"],
+          }}
+          defaultValue={templateId}
+          step="any"
+          errorMessage={
+            errors?.templateId
+              ? { children: ["This field is required"] }
+              : undefined
```

Comment:
maybe it is easier to hoist this up into a method? 

```
const errorMessageFor = (id, name) = {
  return errors?.[id] ? { children: [i18n(..., {field: name})] } : undefined
}
```

Reply:
will do it as part of 844 - as that will require error summaries ,  message will be constructed in validate method